server/src/storage/segment.rs

```diff
@@ -13,16 +13,16 @@
  * limitations under the License.
  */
 
-//! Support for different kinds of uncompressed segments. The SegmentBuilder struct provides support
-//! for inserting and storing data in an in-memory segment. SpilledSegment provides support for
+//! Support for different kinds of uncompressed segments. The [`SegmentBuilder`] struct provides support
```

Comment:
Maybe replace the instances of "uncompressed segment" with "UncompressedSegment" or "types implementing UncompressedSegment".

Reply:
Fixed.